### PR TITLE
EPMRPP-34185 Fixed disabled for 'Delete' filter icon

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.jsx [hunk 1]
```diff
@@ -94,9 +94,9 @@ export const FilterTableItem = ({
       <div className={cx('separator')} />
     </div>
     <div className={cx('block', 'delete-block')}>
-      <div className={cx('bin-icon', { disabled: !canBeDeleted })} onClick={onDelete}>
-        {Parser(BinIcon)}
-      </div>
+      <button className={cx('delete-button')} disabled={!canBeDeleted} onClick={onDelete}>
```
Comment (ArtsiomYeliseyenka) [inline app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.jsx]: What the purpose to wrap div with additional button tag?

Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.jsx]: Now interface component got an outline in focus

Comment (AmsterGet) [reply app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.jsx]: Button is necessary for control disabled status. So <div> just rendered like disabled, but still clickable. Fixed button outline.

Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.jsx]: You can prevent click on a div with disabled status just controlling onClick callback execution.
 something like onClick={!disabled && onDelete}

File: app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.scss [hunk 2]
```diff
@@ -200,6 +200,11 @@
     padding-right: 0;
   }
 }
+.delete-button {
+  background: none;
+  outline: none;
+  border: none;
```
Comment (yana95) [inline app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.scss]: Use this  https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/ order for css rules

File: app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.jsx (hunk 1 above)
Comment (AmsterGet) [reply app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.jsx]: Thank you for suggestion. Fixed

File: app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.scss (hunk 2 above)
Comment (AmsterGet) [reply app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.scss]: Already not actual.

**Pull request merged**